src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -128,11 +128,15 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
       final Password password = config.getPassword(
           ElasticsearchSinkConnectorConfig.CONNECTION_PASSWORD_CONFIG);
 
+      final boolean compression = config.getBoolean(
```

Comment:
nit: Maybe change the variable name to `compressionEnabled` for better clarity, although it's questionable if it's worth it as it's a short-lived variable

Reply:
@stanislavkozlovski, I guess that correct name is `compressionIsEnabled` because it is should be a passive voice because an user is enable an compression.

But I don't think that it is important for this case ;)

Reply:
@stanislavkozlovski I agree with you because fiend name is `requestCompressionEnabled` and it sounds logical to rename this variable.

Reply:
yep absolutely, `compressionIsEnabled` is better

Reply:
@stanislavkozlovski but it requests also to rename a field and getter/setter at jest-http client that looks overengineering for this goal and that also broke back compatibility.

So, I prefere to use incorrect value but similar to name of setter ;)